dangerous warning about SpatiallyVaryingKernel

```
statismo/modules/VTK/examples/vtkSpatiallyVaryingGPModelExample.cxx:153:39: warning: no viable constructor copying variable of type 'SpatiallyVaryingKernel<RepresenterType::DatasetType>'; C++98 requires a copy constructor when binding a reference to a temporary [-Wbind-to-temporary-copy]
        const MatrixValuedKernelType& temperedKernel = SpatiallyVaryingKernel<RepresenterType::DatasetType>(representer, gk, temperingFun , numberOfComponents,  numberOfComponents * 2, true);
                                      ^                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
statismo/modules/core/include/KernelCombinators.h:194:7: note: candidate constructor (the implicit copy constructor) not viable: expects an l-value for 1st argument
class SpatiallyVaryingKernel : public MatrixValuedKernel<typename Representer<T>::PointType> {
```

--- a/modules/VTK/examples/vtkSpatiallyVaryingGPModelExample.cxx
+++ b/modules/VTK/examples/vtkSpatiallyVaryingGPModelExample.cxx
@@ -90,7 +90,7 @@ vtkPoint centerOfMass(const vtkPolyData* _pd) {
     }
     double V = 1.0 / static_cast<double>(numPoints);
     for (unsigned d = 0; d < 3; ++d) {
-        centerOfMass[d] * V;
+        centerOfMass[d] *= V;
     }
     return centerOfMass;
 }

